docs/source/essentials/data.md

```diff
@@ -51,58 +132,111 @@ context: async () => ({
 }
 ```
 
-## Implementing Queries in Apollo Server
+<h2 id="default">Default resolvers</h2>
+
+Explicit resolvers are not needed for every type, since Apollo Server provides a [default](https://github.com/graphql/graphql-js/blob/69d90c601ad5a6f49c06b4ebbc8c73d51ef03566/src/execution/execute.js#L1264-L1278) that can perform two actions depending on the contents of `parent`:
```

Comment:
I don't think we need to link to the default resolver implementation in `graphql-js`.  Perhaps our explanation is enough?